docs/devenv_rhel8.md

```diff
@@ -55,8 +55,10 @@ Run the following commands to configure SUDO, upgrade the system, install basic
 ```bash
 echo -e 'microshift\tALL=(ALL)\tNOPASSWD: ALL' | sudo tee /etc/sudoers.d/microshift
 sudo dnf update -y
-sudo dnf install -y git cockpit make golang selinux-policy-devel rpm-build bash-completion
+sudo dnf install -y git cockpit make gcc selinux-policy-devel rpm-build bash-completion
 sudo systemctl enable --now cockpit.socket
+export GO_VER=1.18.7; curl -L -o go${GO_VER}.linux-amd64.tar.gz https://go.dev/dl/go${GO_VER}.linux-amd64.tar.gz && sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go${GO_VER}.linux-amd64.tar.gz
+echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc && source ~/.bashrc
```

Comment:
If we're not installing 'golang' package, I would rather create a symlink from /usr/local/bin/go pointing to the new package. Let's not create a dependency on custom environment.

Reply:
Done